labelbox/client.py

```diff
@@ -117,19 +126,35 @@ def convert_value(value):
                 value = value.strftime("%Y-%m-%dT%H:%M:%SZ")
             return value
 
-        if params is not None:
-            params = {
-                key: convert_value(value) for key, value in params.items()
-            }
-
-        data = json.dumps({'query': query, 'variables': params}).encode('utf-8')
-
+        if query is not None:
+            if params is not None:
+                params = {
+                    key: convert_value(value) for key, value in params.items()
+                }
+            data = json.dumps({
+                'query': query,
+                'variables': params
+            }).encode('utf-8')
```

Comment:
If `params` is `None`, would it set the value of `variables` to `undefined`? Is that OK here?

Reply:
Yeah this should be fine. Here is a minimal example you can try that works without setting params.
```
from labelbox import Client
client = Client()
client.execute(""" query { organization {id} }""")
```